### DISCUSSION TITLE
Comboboxes enhancements !

### DISCUSSION THREAD

**Discussion by flarive** [flarive/Neumorphism.Avalonia#8] [Announcements]
I made several enhancements on ComboBoxes :
- Outset style
- Possible left or right header
- Few alignements fixes
- Nicer comboxes in demo page

<hr /><em>This discussion was created from the release <a href='https://github.com/flarive/Neumorphism.Avalonia/releases/tag/3.0.1'>Comboboxes enhancements !</a>.</em>

**Comment by ApaulMyLittleAirport** [comment]
Wow! It was really amazing. I'm also in Wow! It was really amazing. I'm also using avalonia and this library is great.

**Reply by flarive** [reply]
Thanks a lot for this cool feedback :)
